test/artefacts/estimate-self-assessment-penalties/y.txt

```diff
@@ -8,6 +8,7 @@ For which tax year do you want the estimate?
   * 2012-13: 6 April 2012 to 5 April 2013
   * 2013-14: 6 April 2013 to 5 April 2014
   * 2014-15: 6 April 2014 to 5 April 2015
+  * 2015-16: 6 April 2015 to 5 April 2016
```

Comment:
Would it be worth to add `2015-2016` to questions and responses and generate the new artefacts?